index.html

```diff
@@ -1,9 +1,173 @@
+          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/4d6054f9-logo-cisco.svg" alt="Cisco">
+      </li>
+      <li class="inline-logos__item">
+          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/a298e7ec-logo-hp.svg" alt="HP">
+      </li>
+      <li class="inline-logos__item">
+          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/f7db330d-partner-logo-chaosprime.png" alt="ChaosPrime">
+      </li>
+      <li class="inline-logos__item">
+          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/3630a703-partner-logo-allwinnertech.png" alt="Allwinner Technology">
+      </li>
+      <li class="inline-logos__item">
+          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/23835286-partner-logo-rti.png" alt="Real-Time Innovations">
+      </li>
+      <li class="inline-logos__item">
+          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/e6bc8a26-partner-logo-osrf.png" alt="OSRF">
+      </li>
+      <li class="inline-logos__item">
+          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/d27eee64-partner-logo-banana-pro.png" alt="LeMaker">
+      </li>
+    </ul>
+  </div>
+</section>
+
+<!-- Source Code -->
+<section class="row">
+  <div class="twelve-col">
+    <h2>Source Code</h2>
+
+    <div class="four-col">
+      <div>Browser and branch from Bazaar</div>
+    </div>
+
+    <div class="four-col">
+      <div>Browse, branch, fork or clone from Github</div>
+    </div>
+
+    <div class="four-col">
+      <div>Download the offically released archives</div>
+    </div>
+  </div>
+</section>
+
+<!-- Public Cloud Examples -->
+<section class="row">
+  <div class="twelve-col">
+    <h2>Used across the public cloud</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. In qui adipisci
+      assumenda, enim ipsum consequatur exercitationem, temporibus laudantium
+      beatae unde ut molestiae, corporis? Quos adipisci, provident, vel sed non.
+    </p>
+
+  </div>
+</section>
+
+<!-- Support channels -->
+<section class="row">
+
+  <div class="twelve-col">
+
+    <h2>Need support?</h2>
+
+    <a href="https://bugs.launchpad.net/cloud-init/+filebug">Report a bug</a>
+    <a href="http://pingx.net/freenode/cloud-init">Chat on freenode</a>
+    <a href="http://cloudinit.readthedocs.org/">Read the docs</a>
+  </div>
+</section>
+
+<!-- About / The Team -->
+<section class="row">
+
+  <div class="six-col">
+
+    <h2>About</h2>
+
+    <p>cloud-init initializes instances in cloud environments. cloud-init is
+      developed and released as free software under the
+      <a href="http://www.gnu.org/licenses/gpl-3.0.en.html">GPLv3 open source
+      license</a>. Originally designed for the Ubuntu distribution of Linux in
+      Amazon EC2, cloud-init is now supported on many Linux and UNIX
+      distributions in every major cloud.
+    </p>
+  </div>
+
+  <div class="six-col">
```

Comment:
I plan to submit a PR to upstream vanilla-framework removing last col and replacing with;

```
[class*=-col]:last-child {
  margin-right: 0;
}
```

..as this will remove the need to add an extra class?